sql/sys_vars.cc

```diff
@@ -5073,6 +5073,33 @@ static Sys_var_ulong Sys_host_cache_size(
        NO_MUTEX_GUARD, NOT_IN_BINLOG, ON_CHECK(NULL),
        ON_UPDATE(fix_host_cache_size));
 
+static Sys_var_ulong Sys_keepalive_time_seconds(
+       "keepalive_time",
+       "The interval between the last data packet sent and the first keepalive probe.",
+       AUTO_SET GLOBAL_VAR(keepalive_time),
+       CMD_LINE(REQUIRED_ARG), VALID_RANGE(0, 65536),
+       DEFAULT(KEEPALIVE_TIME),
+       BLOCK_SIZE(1),
+       NO_MUTEX_GUARD, NOT_IN_BINLOG, ON_CHECK(NULL));
```

Comment:
Have you considered an ON_UPDATE function to change existing connections? (On all vars)